<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Discrete Choice Model Builder</string>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="layout_centralwidget">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="box_columns">
       <property name="title">
        <string>Columns</string>
       </property>
       <layout class="QVBoxLayout" name="layout_box_columns"/>
      </widget>
      <widget class="QGroupBox" name="box_config">
       <property name="title">
        <string>Configuration</string>
       </property>
       <layout class="QVBoxLayout" name="layout_box_config">
        <item>
         <widget class="QToolBox" name="toolbox_config">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="page_model">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>773</width>
             <height>401</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Model</string>
           </attribute>
           <layout class="QVBoxLayout" name="layout_page_model"/>
          </widget>
          <widget class="QWidget" name="page_process">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>773</width>
             <height>401</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Processing</string>
           </attribute>
           <layout class="QVBoxLayout" name="layout_page_process"/>
          </widget>
          <widget class="QWidget" name="page_eval">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>773</width>
             <height>401</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Evaluation</string>
           </attribute>
           <layout class="QVBoxLayout" name="layout_page_eval"/>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="action_undo"/>
    <addaction name="action_redo"/>
    <addaction name="separator"/>
    <addaction name="action_cut"/>
    <addaction name="action_copy"/>
    <addaction name="action_paste"/>
    <addaction name="action_delete"/>
    <addaction name="separator"/>
    <addaction name="action_find"/>
    <addaction name="separator"/>
    <addaction name="action_select_all"/>
   </widget>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="action_new_project"/>
    <addaction name="action_project_open"/>
    <addaction name="action_project_save"/>
    <addaction name="action_project_save_as"/>
    <addaction name="separator"/>
    <addaction name="action_import_data"/>
    <addaction name="action_export_data"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
  </widget>
  <action name="action_copy">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="action_paste">
   <property name="icon">
    <iconset theme="edit-paste">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="action_cut">
   <property name="icon">
    <iconset theme="edit-copy">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="action_undo">
   <property name="icon">
    <iconset theme="edit-undo">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="action_redo">
   <property name="icon">
    <iconset theme="edit-redo">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="action_select_all">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="action_find">
   <property name="icon">
    <iconset theme="edit-find">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_new_project">
   <property name="icon">
    <iconset theme="document-new">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>New Project ...</string>
   </property>
  </action>
  <action name="action_project_open">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Open Project ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_project_save">
   <property name="icon">
    <iconset theme="document-save">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_project_save_as">
   <property name="icon">
    <iconset theme="document-save-as">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Save Project As ...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_import_data">
   <property name="icon">
    <iconset theme="document-page-setup">
     <normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</normaloff>../../../../../../../../../../Universität/Semester/2023 SS/Veranstaltungen/Praxis der Softwareentwicklung/Projekt/Discrete Choice Model Builder/src/view/ui</iconset>
   </property>
   <property name="text">
    <string>Import Data</string>
   </property>
  </action>
  <action name="action_export_data">
   <property name="icon">
    <iconset theme="document-send">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Export Data</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
